docs/getting_started_tor.md

```diff
@@ -9,28 +9,58 @@ This example requires that you have installed:
 * [`tor`](https://github.com/torproject/tor/blob/main/README); can typically be installed
 via your system package manager.
 * [`tgen`](https://github.com/shadow/tgen); will most likely need to be built from source.
-* [`obfs4proxy`](https://gitlab.com/yawning/obfs4); can typically be installed via your system package manager. The simulation will still functon without it, but the simulated
-hosts `ptbridge` and `torptbridgeclient` will have errors.
 
-Once Shadow, Tor, and TGen are installed, you can quickly get started running a
-very simple Tor network:
+## Configuring Shadow
+
+This simulation again uses `tgen` as both client and server. In addition to a
+`tor`-oblivious client and server, we add a `tor` network and a client that uses
+`tor` to connect to the server.
+
+`shadow.yaml`:
+
+```yaml
+{{#include ../examples/docs/tor/shadow.yaml}}
+```
+
+## Running the Simulation
+
+We run this example similarly as before. Here we use an additional command-line
+flag `--template-directory` to copy a template directory layout containing each
+host's `tor` configuraton files into its host directory before the simulation
+begins.
```

Comment:
Maybe add a link to where they can find this template directory if they want to run it on their own?

Reply:
Done